build/yamls/antrea.yml

```diff
@@ -38,6 +38,118 @@ spec:
     served: true
     storage: true
 ---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
We should not make yaml changes until the feature is ready end-to-end?

Reply:
Yes, otherwise we will have to deal with cleanup if anything is changed.
Maybe add parameter to kustomization so manifest with ClusterNetworkPolicy can be generated manually for developers.

Reply:
added a new mode --cnp for devs to generate manifest .. PTAL